### PR TITLE
Model Building and Hyperparameter tuning

### PR DESCRIPTION
**Pull request by Adelakun1999**
I experimented with different models on the dataset , models such as : 

1. Linear Regression
2. Random Forest 
3. Decision Tree
4. Gradient Boosting 
5. XGBOOST 

Model hyperparater tuning was done on Gradient Boosting using Grid Search and Hyperopt approach was utilized to tune the Xgboost the model. 

The model was evaluated using rmse, mse and mae 

A PR against Issue [4](https://github.com/mlsanigeria/nigeria-crime-trends/issues/4) and [5](https://github.com/mlsanigeria/nigeria-crime-trends/issues/5) has  been made.

### CODE REVIEW

**Comment by Adelakun1999** [comment]
@Sammybams can you kindly check out my contribution

**Review by Sammybams — APPROVED** [review]
Great work @Adelakun1999. Thank you very much for your contribution especially the feature importance which can be further used in an analysis.
